source/openapi3/split.yaml

```
@@ -4142,7 +4142,7 @@ components:
           example: '9/13/2016 12:00:00 AM'
         your_bank_account_id:
           type: string
-          description: Refers to `initiator_bank_account_id`. You can use this to identify issues related to the initiator's bank account.
+          description: Specify where we should take the funds this transaction. If omitted, your primary bank account will be used.
```

Like @trev suggested

```suggestion
          description: Specify where we should settle the funds for this transaction. If omitted, your primary bank account will be used.
```